### PR TITLE
Delegate `to_sentence` and `to_fomatted_s` to `records`

### PR DESCRIPTION
**Pull request by kenta-s**
Fix: https://github.com/rails/rails/issues/27901

### CODE REVIEW

File: activerecord/lib/active_record/relation/delegation.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def inherited(child_class)
     # for each different klass, and the delegations are compiled into that subclass only.
 
     delegate :to_xml, :encode_with, :length, :collect, :map, :each, :all?, :include?, :to_ary, :join,
-             :[], :&, :|, :+, :-, :sample, :reverse, :compact, :in_groups, :in_groups_of,
+             :[], :&, :|, :+, :-, :sample, :reverse, :compact, :in_groups, :in_groups_of, :to_sentence, :to_formatted_s, :to_xml,
```
Comment (simi) [inline activerecord/lib/active_record/relation/delegation.rb]: Maybe wrap to next line?

**Comment by kenta-s** [comment]
@simi Thank you. And I mistakenly added `to_xml` twice(which was working correctly) so I removed it.

**Comment by maclover7** [comment]
@kenta-s Can you add a test to confirm the new behavior? Thanks!

**Comment by simi** [comment]
Should be enough to add new delegations to this [array](https://github.com/rails/rails/blob/d13bc5df902a2c82c3096b627830be97acbedf50/activerecord/test/cases/relation/delegation_test.rb#L29-L36) in test suite.

**Comment by kenta-s** [comment]
done :)

**Comment by eileencodes** [comment]
Backported to 5-0-stable in 28a1bdd6fde91781f764677e5178ab17b8439060

**Pull request merged**